### DISCUSSION TITLE
Docs build fails when using insiders

### DISCUSSION THREAD

**Discussion by SteveRosam** [squidfunk/mkdocs-material#7497] [General]
Weve been using insiders without issue for a long while.
3 days ago we got a build error

```
Run pip install git+https://***@github.com/squidfunk/mkdocs-material-insiders.git
Collecting git+https://****@github.com/squidfunk/mkdocs-material-insiders.git
  Cloning https://****@github.com/squidfunk/mkdocs-material-insiders.git to /tmp/pip-req-build-mmlocueo
  Running command git clone --filter=blob:none --quiet 'https://****@github.com/squidfunk/mkdocs-material-insiders.git' /tmp/pip-req-build-mmlocueo
  remote: Repository not found.
  fatal: repository 'https://github.com/squidfunk/mkdocs-material-insiders.git/' not found
  error: subprocess-exited-with-error
```

Has anything changed with using the insiders version?

I checked the GH_TOKEN (hasnt expired) and I generated a new one - same result.

Going to the URL results in a 404: `https://my_token@github.com/squidfunk/mkdocs-material-insiders.git`

Any help appreciated.

**Comment by kamilkrzyskow** [comment]
Hello @SteveRosam,
my classic personal token works for me with this approach in my workflow:

```
https://$GITHUB_ACTOR:$PRIVATE_ACCESS@github.com/squidfunk/mkdocs-material-insiders.git
```

Checking the URL in a private tab won't work as the browser won't connect to the server with the git protocol, and simply connect via the HTTP protocol and load the website, which operates on your actual auth login token in GitHub.

While being logged-in, in your account where you sponsor the project simply access the repository directly to check privileges:
https://github.com/squidfunk/mkdocs-material-insiders

If despite using the correct account you can't access the repository, then your Insiders access has expired, and you're not marked as a Sponsor anymore. 

If you believe there is a mistake, please use the correct contact funnels:
https://squidfunk.github.io/mkdocs-material/support/#sponsoring-support
